src/librustdoc/html/render/print_item.rs

```diff
@@ -1474,22 +1474,17 @@ fn item_constant(w: &mut Buffer, cx: &mut Context<'_>, it: &clean::Item, c: &cle
             typ = c.type_.print(cx),
         );
 
-        // FIXME: The code below now prints
-        //            ` = _; // 100i32`
-        //        if the expression is
-        //            `50 + 50`
-        //        which looks just wrong.
-        //        Should we print
-        //            ` = 100i32;`
-        //        instead?
-
         let value = c.value(tcx);
         let is_literal = c.is_literal(tcx);
         let expr = c.expr(tcx);
         if value.is_some() || is_literal {
```

Comment:
I think this surrounding it should be removed - the `else` will never be hit right now

Reply:
Yeah perfect. Got rid of the outer if